cdc/owner/async_sink.go

```diff
@@ -46,82 +47,99 @@ type AsyncSink interface {
 	// the DDL event will be sent to another goroutine and execute to downstream
 	// the caller of this function can call again and again until a true returned
 	EmitDDLEvent(ctx cdcContext.Context, ddl *model.DDLEvent) (bool, error)
-	SinkSyncpoint(ctx cdcContext.Context, checkpointTs uint64) error
+	EmitSyncPoint(ctx cdcContext.Context, checkpointTs uint64) error
 	Close(ctx context.Context) error
 }
 
 type asyncSinkImpl struct {
-	sink           sink.Sink
-	syncpointStore sink.SyncpointStore
+	lastSyncPoint  model.Ts
+	syncPointStore sink.SyncpointStore
 
-	checkpointTs model.Ts
-
-	lastSyncPoint model.Ts
-
-	ddlCh         chan *model.DDLEvent
+	checkpointTs  model.Ts
 	ddlFinishedTs model.Ts
 	ddlSentTs     model.Ts
 
+	ddlCh chan *model.DDLEvent
+	errCh chan error
+
+	sink            sink.Sink
+	sinkInitHandler asyncSinkInitHandler
```

Comment:
Could you please add a comment to indicate that the purpose of this field is to facilitate unit testing.